Update README.md

felipemarques opened the pull request.

felipemarques committed:
Update README.md

iamluc commented on README.md:
````diff
@@ -43,10 +43,16 @@ $ eval $(docker-machine env mybox)
 $ sudo route -n add 172.0.0.0/8 $(docker-machine ip $(docker-machine active))
 ```
 
+For Windows ( requires admin elevation )
````
You should create a new section like for `Linux` or `Mac OS` or update the Mac OS section

felipemarques replied:
ok ... sorry

iamluc commented on README.md:
```diff
@@ -43,10 +43,16 @@ $ eval $(docker-machine env mybox)
 
 Also, you should add a route to access containers inside your VM.
 
+For Linux
```
For Mac OS

felipemarques replied:
ok sorry

felipemarques committed:
Update README.md

felipemarques committed:
update readme

fix wrong route to public ips 172.x 
now using a private subnet range route /P add 172.17.0.0/8 192.168.99.100

iamluc commented:
Thank you @felipemarques !
